NUModules/NUModule.j

```diff
@@ -1140,7 +1142,13 @@ NUModuleTabViewModeIcon                = 2;
     if (!_isProcessingPush)
         [self showLoading];
 
-    if ([[self class] automaticContextManagement])
+    if (_usesPagination && [_categories count])
```

Comment:
I am not sure we need `_usesPagination` anymore. I think this can be removed as we will always have pagination. What do you think ?